### ISSUE TITLE
Chunk lastal input

### ISSUE THREAD

**Issue by haydenm** [broadinstitute/viral-ngs#318]
Lastal takes too long or fails when the input size (number of reads) on one lane is too large, e.g., ~100M. It would be helpful if reads from a lane were split into chunks and lastal were executed on each, perhaps run in parallel. Chunks of size ~100K should work well. The taxonomic filtered reads for the lane would then be the union across the individual outputs.

**Comment by tomkinsc** [comment]
This should hopefully be improved now that #319 has been merged into master.

**Comment by dpark01** [comment]
dummy comment testing github-slack integration on issue comments
